cmds/core/uptime/uptime.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Get the time the machine has been up
+//
+// Synopsis:
+//     uptime
+//
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// This function takes in the filepath to a file containing loadavgs,it then extracts and returns the three load averages
+func loadavg(filepath string) (load1, load2, load3 string, err error) {
+	loadaverages, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return "-1", "-1", "-1", errors.New("Invalid File Contents")
+	}
+	loadAvgArray := strings.Split(string(loadaverages), " ")
+	return loadAvgArray[0], loadAvgArray[1], loadAvgArray[2], nil
+}
+
+// This function takes in the filepath to a file containing the uptime,it then extracts and returns the uptime in the formart
+// Days , Hours , Minutes ,Seconds
+func uptime(filepath string) (days, hours, minutes int, err error) {
```

Comment:
you have calls to ioutil.ReadFile in two functions. If you make uptime and loadavg take strings, they are easier to test and you can do the ReadFile in one common function. Just a thought.

Reply:
or just use my change above, but let's talk monday.